Dockerfile-sysl2-ubuntu

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
+
+RUN apt-get -qq update -y --fix-missing && \
+  apt-get -qq upgrade -y --fix-missing && \
+  apt-get -qq install -y python2.7 python-pip
+  
+WORKDIR /sysl
+
+ADD dist/sysl-*.whl /sysl
+ADD gosysl/gosysl-linux /sysl
+
+RUN pip install sysl-*.whl
+RUN ln -s /sysl/gosysl-linux /usr/bin/syslgen
```

Comment:
we could rename the `gosysl-*` binaries to `sysl2`.
And then add `sysl2` in the path

Reply:
why not just copy them to the `/usr/bin`?

Reply:
Agreed. Renamed as sysl2 and copied to /usr/bin